### PR TITLE
APEXCORE-735 Upgrade maven-dependency-plugin

### PR DESCRIPTION
**Pull request by vrozov**
@tweise @PramodSSImmaneni Please review

### CODE REVIEW

**Comment by vrozov** [comment]
@tweise @PramodSSImmaneni Please review

**Comment by vrozov** [comment]
testing apache jenkins

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by vrozov** [comment]
another jenkins test

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Apex_Core_PR/848/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Apex_Core_PR/849/

**Comment by pramodin** [comment]
@vrozov the tests seem to be succeeding, why is asfgit asking admins to verify. Is it an automated message. Haven't seen this before.

**Comment by vrozov** [comment]
@PramodSSImmaneni There was a change on the Apache Jenkins side that caused the change to the Jenkins plugin that Apex uses for PR trigger. Please see my email to dev@apex. The message "Can one of the admins verify this patch?" should be suppressed now, as it was caused by empty whitelist on the new Jenkins PR plugin.

**Comment by vrozov** [comment]
test this please

File: apex-app-archetype/src/main/resources/archetype-resources/pom.xml [hunk 1]
```diff
@@ -99,7 +99,7 @@
                </descriptors>
                <archiverConfig>
                  <defaultDirectoryMode>0755</defaultDirectoryMode>
-               </archiverConfig>                  
+               </archiverConfig>
```
Comment (pramodin) [inline apex-app-archetype/src/main/resources/archetype-resources/pom.xml:102]: Can you avoid this and the other formatting changes. They could be made in a separate PR.

Comment (vrozov) [reply apex-app-archetype/src/main/resources/archetype-resources/pom.xml:102]: It is automated IDE setting that strips trailing white spaces.

**Comment by pramodin** [comment]
Can you rebase

**Comment by vrozov** [comment]
done
